roadmaps/2022.md

```diff
@@ -70,13 +70,54 @@ Regarding this point, more details are available on the product team roadmap, bu
+- [X] Release v0.27.0 including indexation speed improvements, the nested field support, the typo tolerance customization, the stabilization/improvement of the cropping & highlighting.
+- [X] Keep being more and more transparent regarding the pre-release tasks each developer makes: write down regularly on GitHub the context, the hypotheses, the results & conclusions, the graphs, the metrics...
+- [X] Keep improving the indexation speed: work with constraints, add benchmarks to prove the change is valuable...
+- [X] v0.28.0: stabilize the API routes. v0.28.0 will be released at the beginning of Q3 but the specs and the whole sprint is during Q2.
+- [X] Keep working on the tokenizer
+  - [X] Clarify the organization regarding the issue management
+  - [X] Make it more contribuable in order to let the community add their language support easily
+  - [X] Publish the tokenizer to crates.io
+- [ ] *Started in Q2* - Autobatching feature: stabilize it, or at least start to work on the stabilization.
+- [ ] *Delayed* Automate our release process by deploying releasemops 🤖
+
+## Q3 2022
+
+The core team has two main goals for this quarter: to improve the product and also include the future release of v1 in our decisions.
+
+### Road to v1
+
+Since API stabilization has been done in v0.28.0, the core team is close to releasing v1. The following list still needs to be done:
+
+- [ ] Stabilize the auto-batching feature: this feature works in a completely transparent way, thus making it not visible to the users.
+- [ ] Clean and stabilize the Meilisearch CLI.
+- [ ] Ensure the API key management feature is stabilized.
+
+v1 will not be released during Q3, but we hope to be able to release it by the end of the year or by the beginning of 2023.
+
+### Improve the Meilisearch product
+
+This part is tightly related to the product team and is mostly decided on their side via the Product Squads and user discussions.
+
+- [ ] Release v0.28.0 (11th July) containing the API stabilization. Handle the support and the user feedback following the massive breaking changes.
+- [ ] Release v0.29.0 (September)
+  - [ ] Fixing user pain points like pagination.
+  - [ ] Improving the document deletion speed.
+  - [ ] Adding really demanded features like multi-index search.
+- [ ] Start v0.30.0 implementation (content not planned yet)
+
+### Team
+
+- [ ] Finish the recruitment process to hire a senior software engineer.
+- [ ] Onboard the new software engineer.
+- [ ] Adapt the team synchronization and rhythm according to the Product Squad (PS) freshly introduced by the product team.
+
+### Misc
+
+- [ ] Finish internship work on Jayson. Integrate it to one or two routes on the Meilisearch side.
+- [ ] Work with the devrel team to promote the contributions we need on the tokenizer side.
+- [ ] Keep improving indexation speed/crash
+  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement.
```

Comment:
I would really to hear about the indexation crashes, it looks like there is no more issue with that!

Reply:
Tell me if I'm wrong @gmourier but the cloud team is still complaining about it (impossible to use small machines) and nothing in the documentation to warn the opensource users

Reply:
@meilisearch/cloud-team Did you experience crashes on the smallest plan?

> nothing in the documentation to warn the opensource users to say "you should use this size of machine"

Yes, we do not recommend anything at this time. The only thing I found is that they recommend increasing the RAM size in case of a memory crash.

Reply:
@gmourier Indeed, we experienced crashes with machines < 2Gb of RAM, even if we set the `MEILI_MAX_INDEXING_MEMORY` variable.
Currently, our smallest plan (1Gb RAM) is actually limited to the consumption of 2.5Gb of RAM in order to avoid crashes.

Issue: https://github.com/meilisearch/meilisearch-cloud/issues/471